params/network_upgrades.go

```diff
@@ -156,7 +156,9 @@ func verifyWithDefault(configTimestamp *uint64, defaultTimestamp *uint64) error
 		return errCannotBeNil
 	}
 
-	if *configTimestamp < *defaultTimestamp {
+	// Allow scheduling of UnscheduledActivationTime (or later) to an earlier time.
+	// This is to allow for the network upgrades to be scheduled in tests.
+	if *defaultTimestamp < uint64(upgrade.UnscheduledActivationTime.Unix()) && *configTimestamp < *defaultTimestamp {
```

Comment:
in the updated avalanchego, etna is rescheduled on defaults to UnscheduledActivationTime, which means we can't use Etna at genesis unless we use a timestamp past UnscheduledActivationTime, which seems weird.

so it seems more logical that we could override it if it was `UnscheduledActivationTime`, for testing purposes or local testing.

Reply:
I took another route by changing the test snow context here: https://github.com/ava-labs/subnet-evm/pull/1348/files